redirects.js

```diff
@@ -12,4 +12,18 @@ module.exports = {
   '/about/trademark/': 'https://trademark-policy.openjsf.org/',
   // Redirects the old `/docs` path to `/api`
   '/docs/*': '/api/',
+  '/en/learn/the-packagejson-guide/': '/en/learn/',
```

Comment:
I don't think we need the language prefixes. Afaik we generate locale redirects for each redirect (https://github.com/nodejs/nodejs.dev/blob/main/gatsby-node.js#L194)

But not sure if it's the same for the "source" part of link

Reply:
I got these links by doing `site:nodejs.dev learn ` in google. I’m also not sure if the prefix is needed.